assets/src/dashboard/app/views/editorSettings/publisherLogo/stories/index.js

```diff
@@ -25,5 +30,13 @@ export default {
 };
 
 export const _default = () => {
-  return <PublisherLogoSettings />;
+  return (
+    <PublisherLogoSettings
+      onUpdatePublisherLogo={(e) => {
+        e.preventDefault();
```

Comment:
Will we need to `preventDefault` for all usages of this logo? If so we could just bubble this up and then wrap the callback.

Reply:
nah this is just because there's nothing there yet since the fileUpload is separate and i hate it when storybook clicks send you out of the existing view.